setup.py

```diff
@@ -28,7 +68,7 @@
     },
     install_requires=[
         "OpenTimelineIO >= 0.12.0",
-        "shotgun_api3 @ git+https://github.com/shotgunsoftware/python-api.git",
+        "shotgun-api3 >= 3.3.3",
```

Comment:
How does that work, since shotgun-api3 is not available on Pypi, only github?

Reply:
Why not add the `>=` and keep the `@`? Isn't it possible?
If some dependency needs to be installed, it needs to be in the installation README.

Reply:
And how is the CI machine installing this if it has no access to it?

Reply:
not possible, should be added to the README, yes. I'll add a polish ticket for all of that.

Reply:
I added `pip install git+https://github.com/shotgunsoftware/python-api.git` to the CI